### DISCUSSION TITLE
Can we control visibility of user feild other then email

### DISCUSSION THREAD

**Discussion by MuhammadKhizar7** [pocketbase/pocketbase#3541] [Q&A]
For example user have phone number filed can control visibility like in email in user so user can make it public or private

**Comment by ganigeorgiev** [comment]
No, at least not out of the box. Also note that users have control over the visibility of their own email and can modify the field even if it is set to `Public: off`.

If you want to restrict access (including modify operations) to a specific field, you can move them to a new collection.

**Reply by MuhammadKhizar7** [reply]
It can be potential feature to allow user make a field set public or private 

> If you want to restrict access (including modify operations) to a specific field, you can move them to a new collection.

No I am not ask restrict access to modify just make private only or public
